llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -1104,6 +1104,10 @@ def FeatureStdExtP
 def HasStdExtP : Predicate<"Subtarget->hasStdExtP()">,
                  AssemblerPredicate<(all_of FeatureStdExtP),
                                     "'Base P' (Packed SIMD)">;
+def FeatureEnablePExtCodeGen
```

Comment:
I didn't mean to add this here. I meant to add it as a cl::opt to RISCVSubtarget.cpp. Then manually add `enablePExtCodeGen()` to RISCVSubtarget." That enablePExtCodeGen() should check the cl::opt and hasStdExtP(). So all of the code only needs to check `enablePExtCodeGen()` instead of separately checking both.